lighthouse-cli/chrome-finder.ts

```diff
@@ -133,3 +182,27 @@ function canAccess(file: string): Boolean {
     return false;
   }
 }
+
+function uniq(arr: Array<any>) {
+    return arr.sort().filter((item: any, pos: number, _this: any) =>
+      !pos || item != _this[pos - 1]
+    );
+}
+
+function findChromeExecutables(desktopPath: string): Array<string> {
```

Comment:
naming: this doesnt seem to be a desktopPath anymore, eh? is it just folderPath?